media/js/helpfulness.js

```diff
@@ -0,0 +1,84 @@
+(function(win, doc, $) {
+    // no localStorage, no helpfulness rating
+    if (('localStorage' in win)) {
+        var stfu = localStorage.getItem('helpful-stfu') === 'true'; // true if ever clicked stfu
+        var asked_recently = parseInt(localStorage.getItem(doc.location + '#answered-helpful')) > Date.now();
+        if (!stfu && !asked_recently) {
+
+            setTimeout(function() {
```

Comment:
Big anonymous functions make the code hard to follow -- I recommend making this a named function and using that as the first argument to `setTimeout`.

```
setTimeout(myFn, 60000);

function myFn() {
//... body
}
```